profiling/proto/service/service.proto

```diff
@@ -0,0 +1,91 @@
+// Copyright 2019 gRPC authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+// This package defines the proto messages and RPC services exposed by gRPC for
+// profiling management. A reference client implementation to interact with
+// this service is provided as a command-line application. This service can be
+// used to toggle profiling on and off and retrieve stats from a gRPC
+// application.
+option go_package = "google.golang.org/grpc/profiling/proto/service";
+
+// EnableRequest defines the fields in a /Profiling/Enable method request to
+// toggle profiling on and off within a gRPC program.
+message EnableRequest {
+  // Setting this to true will enable profiling. Setting this to false will
+  // disable profiling.
+  bool enabled = 1;
+}
+
+// EnableResponse defines the fields in a /Profiling/Enable method response.
+message EnableResponse {
```

Comment:
Maybe delete this type and use https://github.com/protocolbuffers/protobuf/blob/master/src/google/protobuf/empty.proto

It's unlikely we'll ever want to add data here.

Reply:
Unlikely, but we can't guarantee that forever (for example, enabling might fail in the future for some reason we don't have right now?). Personally not a fan of `empty.proto` (especially in a strongly typed languages like Go) because they don't allow changing the response type later even though the on-wire representation will be unchanged. Given that we want to allow clients other profiling/cmd to control profiling, their .pb.gos might break if we decide to change this in the future. I propose we stick to using our own empty response message.

Reply:
@easwars mentioned this earlier too FWIW.